ios/PurchasesHybridCommon/PurchasesHybridCommonIntegrationTests/BaseIntegrationTests.swift

```diff
@@ -77,7 +77,7 @@ private extension BaseIntegrationTests {
 
     private var dangerousSettings: DangerousSettings {
         return .init(autoSyncPurchases: true,
-                     internalSettings: .init(enableReceiptFetchRetry: true))
+                     internalSettings: DangerousSettings.Internal(enableReceiptFetchRetry: true))
```

Comment:
I fixed this (changed in https://github.com/RevenueCat/purchases-ios/pull/2574).